### PR TITLE
Tweak progress bar logger.

### PR DESCRIPTION
**Pull request by knighton**
Isn't this prettier?

```
    0 train 100%|█████████████████████████| 48/48 [00:10<00:00,  4.74it/s, loss/train=2.2352]                           
    1 val   100%|█████████████████████████| 10/10 [00:00<00:00, 11.10it/s, metrics/eval/Accuracy=0.2253]                
    1 train 100%|█████████████████████████| 48/48 [00:09<00:00,  4.99it/s, loss/train=1.1940]                           
    2 val   100%|█████████████████████████| 10/10 [00:00<00:00, 10.13it/s, metrics/eval/Accuracy=0.5603]                
    2 train 100%|█████████████████████████| 48/48 [00:09<00:00,  4.88it/s, loss/train=0.9715]                           
    3 val   100%|█████████████████████████| 10/10 [00:00<00:00, 10.94it/s, metrics/eval/Accuracy=0.5965]
```

### CODE REVIEW

File: composer/loggers/progress_bar_logger.py [hunk 1]
```diff
@@ -157,10 +161,9 @@ def _start(self, state: State):
         assert self.is_train is not None, "self.is_train should be set by the callback"
         assert state.dataloader_len is not None, "dataloader_len should be set when using tqdm"
 
-        desc = f'Epoch {int(state.timestamp.epoch)}'
+        split = 'train' if self.is_train else 'val'
+        desc = f'{int(state.timestamp.epoch):5d} {split:5s}'
```
Comment (ravi-mosaicml) [inline composer/loggers/progress_bar_logger.py:165]:
```suggestion
        desc = f'Epoch {int(state.timestamp.epoch):5d} {split:5s}'
```

**Review by ravi-mosaicml — APPROVED** [review]
LGTM

File: composer/loggers/progress_bar_logger.py (hunk 1 above)
Comment (knighton) [reply composer/loggers/progress_bar_logger.py]: Done

**Comment by knighton** [comment]
```
Epoch     0 train 100%|█████████████████████████| 48/48 [00:10<00:00,  4.79it/s, loss/train=2.2352]                     
Epoch     1 val   100%|█████████████████████████| 10/10 [00:00<00:00, 10.44it/s, metrics/eval/Accuracy=0.2253]          
Epoch     1 train 100%|█████████████████████████| 48/48 [00:09<00:00,  4.89it/s, loss/train=1.1940]                     
Epoch     2 val   100%|█████████████████████████| 10/10 [00:00<00:00, 11.44it/s, metrics/eval/Accuracy=0.5603]          
Epoch     2 train   8%|██                       | 4/48 [00:01<00:10,  4.25it/s, loss/train=1.2142]  
```

**Pull request merged**